src/odemis/driver/technolution.py

```diff
@@ -985,7 +986,7 @@ def __init__(self, name, role, parent, **kwargs):
         self.filename = model.StringVA("storage/images/date/project/megafield_id", setter=self._setFilename)
         self.dataContent = model.StringEnumerated('empty', DATA_CONTENT_TO_ASM.keys())
         # delay between the trigger signal to start the acquisition, and the start of the recording by the mppc detector
-        self.acqDelay = model.FloatContinuous(0.0, range=(0, 200e-6), unit='s', setter=self._setAcqDelay)
+        self.acqDelay = model.FloatContinuous(0.0, range=(0.0, 1.0), unit='s', setter=self._setAcqDelay)
```

Comment:
I'd suggest to add a comment here that acqDelay depends on scanDelay. If increasing it, scanDelay should be increased first.